augment_manta_vcf/1.0/augment_manta_vcf.py

```diff
@@ -0,0 +1,410 @@
+#!/usr/bin/env python3
+
+"""
+augment_manta_vcf.py
+====================
+
+This script augments Manta VCF files with additional information. Namely:
+    1) TR (FORMAT field): Sum of the SR and PR FORMAT fields per allele.
+    2) DP (FORMAT field): Sum of the SR and PR FORMAT fields across all alleles.
+    3) VAF (FORMAT field): Variant allele fraction for the alternate allele.
+    4) REGIONS (INFO field): List of regions from the given BED files that
```

Comment:
Can you add a brief example use case? I still can't exactly figure out when I would do this.

Reply:
Good suggestion! I added this:
> One example use case is labelling breakpoints occuring in loci of interest such as MYC, BCL2, BCL6, and the three IG loci. This can readily be done downstream, but having the breakpoints labelled in the VCF file can enable quick exploratory analyses.